test/Interop/Cxx/reference/rvalue-reference.swift

```diff
@@ -0,0 +1,54 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs/ -Xfrontend -enable-experimental-cxx-interop -Onone) | %FileCheck %s
+// RUN: %target-run-simple-swift(-I %S/Inputs/ -Xfrontend -enable-experimental-cxx-interop -O) | %FileCheck -check-prefix=CHECK-OPT %s
+//
+// REQUIRES: executable_test
+
+import SpecialMembers
+
+func consume(_ x: consuming MoveOnly) {}
+func consume(_ x: consuming Copyable) {}
+
+func main() {
+    let x = MoveOnly()
+// CHECK: MoveOnly 0 created
+// CHECK-OPT: MoveOnly 0 created
```

Comment:
Can you document why this `CHECK-OPT` needs to exist? Is it because of potentially optimized temporaries?

If that's the case, I'm wondering whether it might be worth also testing this with `-fno-elide-constructors`, to ensure those temporaries are forcibly created.

Reply:
Oh. I just looked at the `// RUN` directives and realized that `OPT` stands for "optimized" and not "optional". (And I thought that `CHECK-OPT` was built into LLVM-lit, rather than being user-defined.)

Disregard my question about what `CHECK-OPT` means, but I'm still wondering about `-fno-elide-constructors`, and I also wonder if there's a better `check-prefix` like `CHECK-ONONE` vs `CHECK-DASHO` that will make it obvious what is being tested.

Reply:
Updated the prefixes. `-fno-elide-constructors` is a Clang flag right? Here, the copies are created within Swift, so I would not expect it to make any difference. I think the main differences between optimized and non-optimized code is coming from the optimized version inlining the function defined in Swift so we no longer need some of the copies.